python/docs/source/reference/pyspark.sql/functions.rst

```diff
@@ -533,6 +533,11 @@ JSON Functions
     json_object_keys
     json_tuple
     parse_json
+    is_variant_null
```

Comment:
nit: let's sort the function names alphabetically

Reply:
Done. It looks like `functions.py/scala` is not strictly sorted/grouped, so I only make `functions.rst` sorted.